jdaviz/configs/imviz/tests/test_orientation.py

```diff
@@ -160,6 +167,24 @@ def test_custom_orientation(self):
 
 class TestDeleteOrientation(BaseImviz_WCS_WCS):
 
+    def test_delete_orientation_multi_viewer(self):
+        lc_plugin = self.imviz.plugins['Orientation']
+        lc_plugin.link_type = 'WCS'
+
+        # Should automatically be applied as reference to first viewer.
+        lc_plugin._obj.create_north_up_east_left(set_on_create=True)
+
+        # This would set a different reference to second viewer.
+        viewer_2 = self.imviz.create_image_viewer()
+        self.imviz.app.add_data_to_viewer("imviz-1", "has_wcs_1[SCI,1]")
+        lc_plugin.viewer.selected = "imviz-1"
+        lc_plugin.orientation.selected = "North-up, East-left"

```

Comment:
```suggestion
        lc_plugin.orientation = "North-up, East-left"
```

I think this too.

https://github.com/spacetelescope/jdaviz/blob/deda388472169d11604db304a369869e2e13d00e/jdaviz/configs/imviz/plugins/orientation/orientation.py#L139-L148